scipy/special/tests/test_precompute_gammainc.py

```diff
@@ -36,19 +40,21 @@ def test_g():
 @check_version(sympy, '0.7')
 def test_alpha():
     # Test data for the alpha_k. See DLMF 8.12.14.
+    rtol = 2e-11 if _is_32bit_platform else 1e-17
```

Comment:
Might be better for this one (and friends) to be `knownfailureif`'s instead? If mpmath isn't going to precompute the coefficients correctly (and the point is to get results that are correctly rounded in double precision) then there should be a warning.